docs/config.rst

```diff
@@ -10,11 +10,15 @@ them in order with the latest one taking more priority:
 * ``c:\scrapyd\scrapyd.conf`` (Windows)
 * ``/etc/scrapyd/conf.d/*`` (in alphabetical order, Unix)
 * ``scrapyd.conf``
-* ``~/.scrapyd.conf`` (users home directory)
+* ``~/.scrapyd.conf`` (user's home directory)
 
 The configuration file supports the following options (see default values in
 the :ref:`example <config-example>`).
 
+.. note::
+    The user's home directory is not expanded for the user that may be specified
+    with the ``--uid`` option, but for the user invoking ``scrapyd``.
+    
```

Comment:
yes, i also use `--nodaemon`. `--debug` also seems to work, but i haven't tested to drop into the pdb.

if passing arguments to twisted wasn't supposed, that should be actual behaviour, including not to show twisted's `--help`.